src/Builder.hs

```diff
@@ -142,24 +162,37 @@ instance H.Builder Builder where
         Configure dir -> return [dir -/- "configure"]
 
         Ghc _ Stage0 -> return []
-        Ghc _ _ -> do
+        Ghc _ stage -> do
             win <- windowsHost
             touchyPath <- programPath (vanillaContext Stage0 touchy)
             unlitPath  <- builderPath Unlit
+            ghcdeps <- ghcDeps stage
             return $ [ ghcSplitPath -- TODO: Make conditional on --split-objects
-                     , inplaceLibPath -/- "ghc-usage.txt"
-                     , inplaceLibPath -/- "ghci-usage.txt"
-                     , inplaceLibPath -/- "llvm-targets"
-                     , inplaceLibPath -/- "platformConstants"
-                     , inplaceLibPath -/- "settings"
                      , unlitPath ]
+                  ++ ghcdeps
```

Comment:
How is `inplaceLibPath -/- "ghc-usage.txt"` different from `ghcSplitPath`, `unlitPath` or `touchyPath`? They all are GHC runtime dependencies, but are treated differently here. Why?

Reply:
Do you mean that we should stick ghcSplitPath, unlitPath & touchyPath part of that `ghcdeps` list?

Reply:
Yes! Or add a comment explaining why `ghcSplitPath` and others are treated differently.